srcpkgs/gocryptfs/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'gocryptfs'
+pkgname=gocryptfs
+version=1.6
+revision=1
+build_style=go
+go_import_path="github.com/rfjakob/gocryptfs"
+#go_package="${go_import_path}/cmd/${pkgname}"
+hostmakedepends="git dep"
+makedepends=""
+depends="fuse"
+short_desc="Encrypted overlay filesystem written in Go"
+maintainer="mustaqim <mustaqim+void@pm.me>"
+license="MIT"
+changelog="https://github.com/rfjakob/gocryptfs#changelog"
+homepage="https://nuetzlich.net/gocryptfs"
+distfiles="https://github.com/rfjakob/${pkgname}/archive/v${version}.tar.gz"
+checksum=cc63e18a4d272a59a80296038dfcae64fadbbda920ee309e6def759559447be0
+
+pre_build() {
+	cd $GOSRCPATH
+	dep ensure
+}
+
+do_build() {
+	$GOSRCPATH/build-without-openssl.bash
+}
+
+post_install() {
+	cd $GOSRCPATH
+	vlicense LICENSE
+	vman ${FILESDIR}/gocryptfs.1
```

Comment:
please do not vendor de manpage

Reply:
do not put the manpage in void-linux/void-packages, either put it in distfiles= or build it.

**Never** close a PR just use `git push -f`

Reply:
@maxice8 I've made the necessary changes but the `man` and `license` calls fail(silently)

```bash
# Template file for 'gocryptfs'
pkgname=gocryptfs
version=1.6
revision=1
wrksrc="${pkgname}_v${version}_src-deps"
build_style=go
go_import_path="github.com/rfjakob/gocryptfs"
hostmakedepends="pandoc man-db"
depends="fuse"
short_desc="Encrypted overlay filesystem written in Go"
maintainer="mustaqim <mustaqim+void@pm.me>"
license="MIT"
changelog="https://github.com/rfjakob/gocryptfs#changelog"
homepage="https://nuetzlich.net/gocryptfs"
distfiles="https://github.com/rfjakob/${pkgname}/releases/download/v${version}/${pkgname}_v${version}_src-deps.tar.gz"
checksum=98ff3bc0497ca195f65fa5912ebab33a0f80aa27205b0b35e73908d23dcefbe6

do_build() {
	$GOSRCPATH/build-without-openssl.bash
}

post_install() {
       # with or without `$GOSRCPATH` nothing gets copied here
	vlicense $GOSRCPATH/LICENSE
	$GOSRCPATH/Documentation/MANPAGE-render.bash
	vman $GOSRCPATH/Documentation/gocryptfs.1
}
```

Reply:
@maxice8 After deleting the entire repo and re-cloning it builds and copies the man & license files. I did run `./xbps-src clean` before and `./xbps-src pkg gocryptfs` but it seems to still use the cached build? In any case this is ready to be pulled